mesa/experimental/cell_space/cell.py

```diff
@@ -191,3 +194,30 @@ def _neighborhood(
             if not include_center:
                 neighborhood.pop(self, None)
             return neighborhood
+
+    # PropertyLayer methods
+    def add_property_layer(self, property_layer: PropertyLayer):
```

Comment:
do add and remove an entire layer belong in a cell?

In my view a cell would just do a get and a set on its coordinates in a specified layer.

Reply:
Having direct access to the PropertyLayer is what makes it performant. You don't want to copy all the values everything. So normally you directly access the layer at your coordinate.

Reply:
Removed those two functions, you can directly access the dicts.